aedes.js

```diff
@@ -44,7 +44,10 @@ function Aedes (opts) {
   this.counter = 0
   this.queueLimit = opts.queueLimit
   this.connectTimeout = opts.connectTimeout
-  this.mq = opts.mq || mqemitter(opts)
+  this.mq = opts.mq || mqemitter({
+    concurrency: opts.concurrency,
+    matchEmptyLevels: true // [MQTT-4.7.1-3]
```

Comment:
Should this be removed? It should be automatically turned on now

Reply:
It is safe to set it as we know it.